src/components/Authentication/Authentication.vue

```diff
@@ -0,0 +1,21 @@
+<template>
+    <div>
+       <router-view/>
+    </div>
+</template>
+
+<style>
+.auth-logo {
+  width: 204px;
+  height: 168px;
```

Comment:
Use 'rem' instead of pixels here. Using rem makes the design a little more flexible and can help when the time comes to make the site responsive